packages/@aws-cdk/aws-apigateway/package.json

```diff
@@ -294,7 +294,8 @@
       "docs-public-apis:@aws-cdk/aws-apigateway.ResourceBase.url",
       "attribute-tag:@aws-cdk/aws-apigateway.LambdaAuthorizer.authorizerArn",
       "attribute-tag:@aws-cdk/aws-apigateway.RequestAuthorizer.authorizerArn",
-      "attribute-tag:@aws-cdk/aws-apigateway.TokenAuthorizer.authorizerArn"
+      "attribute-tag:@aws-cdk/aws-apigateway.TokenAuthorizer.authorizerArn",
+      "docs-public-apis:@aws-cdk/aws-apigateway.AccessLogSetting"
```

Comment:
We usually avoid any additional linter exclusions, without good cause. Please add tsdoc for types that the linter complains about.